dlt/pipeline/typing.py

```diff
@@ -48,20 +49,19 @@ def default_dataset(self, new_value: str) -> None:
         self.DEFAULT_DATASET = new_value
 
     @classmethod
-    def from_services_dict(cls, services: StrAny, dataset_prefix: str) -> "GCPPipelineCredentials":
+    def from_services_dict(cls, services: StrAny, dataset_prefix: str, location: str) -> "GCPPipelineCredentials":
```

Comment:
please add a default for `location: str = "US"` to make it backward compatible with all our examples. this code will be removed in v2 anyway so we can take some shortcuts

Reply:
see the linter output
```
./check-package.sh
poetry run mypy --config-file mypy.ini dlt examples
examples/sync_schema_example.py:4: error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/singer_tap_jsonl_example.py:12: error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/singer_tap_example.py:10: error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/quickstart.py:25: error: Missing positional argument "location" in call to "from_services_dict" of "GCPPipelineCredentials"
examples/google_sheets.py:9: error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/google_drive_csv.py:6[8](https://github.com/scale-vector/dlt/actions/runs/3052952945/jobs/4922951182#step:8:9): error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/demo_example.py:84: error: Missing positional argument "location" in call to "from_services_file" of "GCPPipelineCredentials"
examples/rasa_example.py:3[9](https://github.com/scale-vector/dlt/actions/runs/3052952945/jobs/4922951182#step:8:10): error: Missing positional argument "location" in call to "from_services_dict" of "GCPPipelineCredentials"
Found 8 errors in 8 files (checked [11](https://github.com/scale-vector/dlt/actions/runs/3052952945/jobs/4922951182#step:8:12)4 source files)
make: *** [Makefile:41: lint] Error 1
Error: Process completed with exit code 2.
```